autonomy_software/localization_pkg/src/rtk_gps_serial_reader.py

```diff
@@ -23,6 +24,7 @@ def __init__(self):
         self.serial_port = serial.Serial(SERIAL_PORT, baudrate=BAUD_RATE, timeout=1)
 
         self.pub = rospy.Publisher("gps_info", Gprmc, queue_size=10)
+        self.pub_debug = rospy.Publisher("debug_rtk_gps", Gprmc, queue_size=10)
```

Comment:
This one we don't need, since it publishes the exact same thing as the 'main' topic